Dockerfile-locust

```diff
@@ -0,0 +1,19 @@
+FROM locustio/locust AS builder
+
+WORKDIR /go/src/github.com/stolostron/search-indexer
+CMD pwd && ls
+COPY . .
```

Comment:
Sonar has flagged this in other PRs, but I don't see it here.
Anyway, it's better to limit the files copied to only what's needed.
```suggestion
COPY ./test/ ./test/
```